samples/app-ReporterPlus/README.md

```diff
@@ -0,0 +1,183 @@
+
+![image](https://user-images.githubusercontent.com/80528379/123222066-32d3a180-d4ed-11eb-9b6a-b07860a96073.png)
+
+ * Go to Access Keys on the left panel.
+ 
+ ![image](https://user-images.githubusercontent.com/80528379/123224240-44b64400-d4ef-11eb-977c-e3b1e6b6cd6f.png)
+
+	* Copy and paste the Storage account name and any one Connection String somewhere safe. You will need it in a future steps.
+
+ * Now go to the Resource Sharing (CORS) on the left panel and provide the following details under Blob service
+    * Allowed origins = *
+    * Allowed methods = DELETE, GET, HEAD, MERGE, POST, OPTIONS, PUT
+    * Allowed headers = *
+    * Exposed headers = *
+    * Max age = 86400
+    
+ ![image](https://user-images.githubusercontent.com/80528379/123225085-feadb000-d4ef-11eb-8a87-3ec0a6132e8d.png)
+
+2. Navigate to the Containers in the left Panel and create a Container.
+
+![image](https://user-images.githubusercontent.com/80528379/123222526-a8d80880-d4ed-11eb-9b49-fc04aa56bac9.png)
+
+	* Note the Container Name. You will need it in a future steps.
+
+
+ * Go to Shared Access Token on the left panel of the Container and Generate SAS Token and Url by giving the following Permissions.
+    * Read
+    * Add
+    * Create
+    * Write
+    * Delete
+ 
+ ![image](https://user-images.githubusercontent.com/80528379/123228194-eb501400-d4f2-11eb-9618-3e64531e11a9.png)
+ 
+	* Copy and paste the Blob SAS token somewhere safe. You will need it in a future steps.
+	
+Step 3: Run the app locally 
+====================================
+1. Clone the repository.
+
+  		git clone “https://github.com/pnp/teams-dev-samples.git”
+
+2. If you are using Visual Studio
+ 	- Launch Visual Studio.
+ 	- File -> Open -> Project/Solution.
+ 	- Navigate to samples/app-ReporterPlus.
+ 	- Select ReporterPlus.sln file.
+
+3. Update the appsettings.json files
+      - `MicrosoftAppId` & `TenantId` from the Step-1 Point-2.
+      - `MicrosoftAppPassword` from Step-1 Point-3.
+      - `ServiceUrl` is the URL sent by Teams in the Bot payload in the turnContext.Activity.serviceUrl property, dubug the project to get the URL.
+      - `BaseUrl` is the ngrok url that has been generated for the tunneling service as described in the Step-3 Point-4.
+      - `OriginatorId` from Step-1 Point-8.
+      - `SenderEmail` with the email that you have provided to create an Originator Id in the Step-1 Point-8.
+      - `BlobAccountName` and `BlobConnectionString` from the Step-2 Point-1.
+      - `BlobContainerName` and `BlobSAS_Token` from the Step-2 Point-2.
+
+![image](https://user-images.githubusercontent.com/80528379/123258622-32e59880-d511-11eb-8aa0-f068147c8eb4.png)
+
+4. Run Ngrok to expose your local web server via a public URL. Make sure to point it to your Ngrok URI. For example, if you're using port 3978 locally, run:
+
+		Win: ./ngrok http 3978 -host-header=localhost:3978 -subdomain="contoso"
+		Mac: /ngrok http 3978 -host-header=localhost:3978 -subdomain="contoso".
```

Comment:
This seems incorrect. I use the following command on my Mac`ngrok http 3978 -host-header=localhost:3978`. 
Also, please check for windows as well and update here for both.